src/pages/OperatorsPage/index.tsx

```diff
@@ -9,7 +9,7 @@ import { Layer } from '~/utils/Layer'
 import BecomeOperatorModal from '~/modals/BecomeOperatorModal'
 import DelegateFundsModal from '~/modals/DelegateFundsModal'
 import FundSponsorshipModal from '~/modals/FundSponsorshipModal'
-import CreateSponsorshipModal from '~/modals/CreateSponsorshipModal'
+import CreateSponsorshipModal from '~/network/modals/CreateSponsorshipModal'
```

Comment:
Hm. Don't know if I like having multiple `modals` folders. I've got mixed feelings about superiority of grouping by purpose or functional domain* over grouping by unbiased identity**.

---

*) "It's network related modal".
**) "It's a modal".

Reply:
We can either group folders by the type like:
- src/components
- src/getters
- src/modals
and so on but soon at least one of them will start to be bloated and we will have a hard time finding what we need

Some time ago I suggested having some of the code separated into "parts" like this:
- src/projects/components (actually currently it's /marketplace and there is a mess there)
- src/projects/hooks
- src/network/components
- src/network/hooks

because those "parts" are relatively independent and in my opinion it works - the structure is easy to navigate, you can quickly find what you are looking for. I remember that we were talking about it some time ago

Why did I moved the modal from src/modals to src/network/modals? cause mixing those two approaches mentioned above will create a big mess :) in my opinion, stuff located directly /src should be more global/abstract/shared

I suggest we leave it as I did if you are not agreeing with this fully let's have a discussion about it next week together with Tumppi

Reply:
Let's keep it.

If it becomes chronically inconvenient for me to remember if a hook or component is more projecty than networky, or streamy, then I'll smash it and press on moving it up the tree.

Also, just so you know my reasoning… From subgraph's perspective all is network. That's why it seems inconsistent, the new structure. Look, why `network/operators` and not just `operators` if `streams` and not `network/streams`, eh? Know what I mean?

Anyway, let's leave it. We can adjust. And getting back to this discussion with T. back on board is also in play.

Cheers!